inst/scripts/2.6.0/make-bunis-hspc-metadata.R

```diff
@@ -1,7 +1,7 @@
 write.csv(file="../../extdata/2.6.0/metadata-bunis-hspc.csv",
     data.frame(
         Title = sprintf("Bunis human HSPC %s", c("counts", "colData", "rowData")),
-        Description = sprintf("%s for the Bunis human haematopoietic stem-progenitor single-cell RNA-seq dataset", 
+        Description = sprintf("%s for the Bunis human haematopoietic stem and progenitor single-cell RNA-seq dataset", 
```

Comment:
Note: changing this will need a re-propagation to ExperimentHub, as the manifests there do not update automatically. Compile it to recreate the CSVs and then we'll notify the EHub maintainers that this is altered.

Reply:
It's not that important really.  I'll just revert this.